src-server/main.ts

```diff
@@ -102,6 +103,7 @@ app.post('/account/:recipient_address', async (req, res) => {
         payload: encode(Buffer.from(SPEND_TX_PAYLOAD), Encoding.Bytearray),
         nonce: nonce++,
         verify: false,
+        ttl: parseInt(SPEND_TX_TTL, 10),
```

Comment:
```suggestion
        ttl: +SPEND_TX_TTL,
```
`parseInt` is tolerant to something like "123test", but we don't need it
unary plus is more strict, but it works fine with space chars that may occur
```
> +' 123 \n'
123
```